src/main/java/chess/Board.java

```diff
@@ -122,18 +104,22 @@ private double getSum(List<Piece> pieces, Color color, double point) {
         for (int rank = 1; rank <= BOARD_SIZE; rank++) {
             pieces.add(findPiece(new Position(file, rank)));
         }
-        return pieces;
+        return Collections.unmodifiableList(pieces);
+    }
+
+    private double getSum(List<Piece> pieces, Color color) {
+        return pieces.stream().filter(piece -> piece.getColor() == color)
+                .filter(piece -> !(piece.getType() == Type.PAWN))
+                .reduce(0.0, (result, piece) -> result + piece.getPoint(), Double::sum)
+                + getPawnPoint(pieces, color);
     }
 
-    private double getPawnPoint(List<Piece> pieces) {
+    private double getPawnPoint(List<Piece> pieces, Color color) {
         double pawnPoint = Type.PAWN.getDefaultPoint();
-        int pawnCount = 0;
-        for (Piece piece : pieces) {
-            if (piece.getType() == Type.PAWN) {
-                pawnCount++;
-            }
-        }
-        return pawnCount > 1 ? pawnPoint / 2 : pawnPoint;
+        long pawnCount = pieces.stream()
+                .filter(piece -> piece.getColor() == color && piece.getType() == Type.PAWN)
+                .count();
+        return pawnCount > 1 ? (pawnPoint / 2) * pawnCount : pawnPoint * pawnCount;
```

Comment:
삼항 연산자가 조금 복잡한데 `if` 문을 사용한 후 early return하면 `else` 없이도 깨끗이 구현할 수 있을 듯합니다.

```suggestion
        if (pawnCount > 1) {
            return (pawnPoint / 2) * pawnCount;;
        }

        return pawnPoint * pawnCount;

Reply:
괜히 한 줄로 쓰려다가 복잡해진 것 같습니다. 말씀주신대로 수정하겠습니다:)